src/js/player.js

```diff
@@ -588,6 +588,12 @@ vjs.Player.prototype.onFullscreenChange = function() {
   }
 };
 
+/**
+ * Fired when an error occurred
```

Comment:
This is probably a nitpick, but to be consistent with the other comments, let's go with "Fired when an error occurs"